modules/dropdown/main/theme.scss

```diff
@@ -2,4 +2,5 @@
   background: $background-col;
   box-shadow: 0px 2px 2px 2px $shadow-col;
   border-color: $border-col;
+  margin-top: $spacing * 1px;
```

Comment:
So the spacing is applied as part of the styles as well as part of the javascript separately?